superset-frontend/cypress-base/cypress/integration/dashboard/edit_mode.test.js

```diff
@@ -23,47 +23,55 @@ describe('Dashboard edit mode', () => {
     cy.server();
     cy.login();
     cy.visit(WORLD_HEALTH_DASHBOARD);
-    cy.get('.dashboard-header [data-test=edit-alt]').click();
+    cy.get('[data-test="dashboard-header"]').find('[data-test=pencil]').click();
   });
 
   it('remove, and add chart flow', () => {
     // wait for box plot to appear
-    cy.get('.grid-container .box_plot');
+    cy.get('[data-test="grid-container"]').find('.box_plot');
 
-    cy.get('.fa.fa-trash')
+    cy.get('[data-test="icon-button"]')
```

Comment:
Wondering if this should be more specific. There might be a _lot_ of icon buttons on the page. If we put the data-test attribute on the exact one we want to test, e.g. `[data-test="delete-grid-container-button"]` then it's easier to reason about, and you don't need to worry about the fragility of using a `.last()`